packages/tslint-config/test/rules/blueprint-classes-constants/true/test.tsx.lint

```diff
@@ -1,17 +1,17 @@
 apt-get
 at 4pt-size
 "script-source pt-large apt-get"
-               ~~~~~~~~  [msg]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  [msg]
 `script-source pt-large apt-get`
-               ~~~~~~~~  [msg]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  [msg]
 `${template} pt-large`
-             ~~~~~~~~  [msg]
+~~~~~~~~~~~~~~~~~~~~~~  [msg]
 
 <Button className="my-class pt-large" />
-                            ~~~~~~~~  [msg]
+                  ~~~~~~~~~~~~~~~~~~~ [msg]
```

Comment:
😞 why does this mark the entire string now instead of just the offending portion like it used to? feels like a regression.

Reply:
Unfortunately, the tslint replacement api only lets me provide a replacement for a failed node. Previously, blueprint could fail the string by character positions, but now I have to fail the entire node so that I can replace it. 

Agreed this is worse for seeing the lint errors... but it is better because it gets rid of them!

Reply:
ok i see that.
is it possible to report the failure in one location and the replacement on a wider location? so we could have both worlds.